avatar.go

```diff
@@ -0,0 +1,56 @@
+//
+// Copyright 2021, Pavel Kostohrys
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package gitlab
+
+import (
+	"net/http"
+)
+
+// AccessRequest represents a access request for a user avatars.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/api/avatar.html
+type AvatarOptions struct {
+	Email string `json:"email"`
+	Size  int    `json:"size"`
+}
```

Comment:
These should be pointers, the struct should be places above the `GetAvatar` method and this the struct should be called `GetAvatarOptions`...